### PR TITLE
Use microsoft defaults

### PR DESCRIPTION
**Pull request by NiklasArbin**

### CODE REVIEW

**Review by andrevirdarson — APPROVED** [review]
Hard to review but looks good to me 😅 Just a couple of questions

File: knightbus-applicationinsights/src/KnightBus.ApplicationInsights/ApplicationInsightsExtensions.cs [hunk 1]
```diff
@@ -2,16 +2,17 @@
 using Microsoft.ApplicationInsights.DependencyCollector;
 using Microsoft.ApplicationInsights.Extensibility;
 using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.QuickPulse;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace KnightBus.ApplicationInsights
 {
     public static class ApplicationInsightsExtensions
     {
-        public static IHostConfiguration UseApplicationInsights(this IHostConfiguration hostConfiguration, TelemetryConfiguration telemetryConfiguration)
```
[the remark this answers is not in the document]
Comment (NiklasArbin) [reply knightbus-applicationinsights/src/KnightBus.ApplicationInsights/ApplicationInsightsExtensions.cs:10]: We don't care, but others might. Not everyone uses New Relic

File: knightbus-schedule/src/KnightBus.Schedule/SchedulingPlugin.cs [hunk 2]
```diff
@@ -23,7 +23,7 @@ public SchedulingPlugin(IHostConfiguration configuration)
         public async Task StartAsync(CancellationToken cancellationToken)
         {
             ConsoleWriter.WriteLine($"Starting {nameof(SchedulingPlugin)}");
-            var lockManager = _configuration.SingletonLockManager ?? throw new SingletonLockManagerMissingException();
+            var lockManager = _configuration.DependencyInjection.GetInstance<ISingletonLockManager>();
```
[the remark this answers is not in the document]
Comment (NiklasArbin) [reply knightbus-schedule/src/KnightBus.Schedule/SchedulingPlugin.cs:26]: Yes we should use GetRequired. I'll change it.

[the remark this answers is not in the document]
Comment (NiklasArbin) [reply knightbus-schedule/src/KnightBus.Schedule/SchedulingPlugin.cs:26]: It already was required. Scheduling can't work without the SingletonLockManager

File: knightbus-applicationinsights/src/KnightBus.ApplicationInsights/ApplicationInsightsExtensions.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrevirdarson) [reply knightbus-applicationinsights/src/KnightBus.ApplicationInsights/ApplicationInsightsExtensions.cs:10]: Of course, but more in the sense of do we want to support and maintain something we don't use?

**Pull request merged**